UnityProject/Assets/Scripts/HealthV2/Living/Damage/BodyPartTraumaDamage.cs

```diff
@@ -439,10 +439,16 @@ public void InternalBleedingLogic(AttackType attackType = AttackType.Internal, D
 		/// </summary>
 		private void DismemberBodyPartWithChance()
 		{
-			float chance = UnityEngine.Random.RandomRange(0.0f, 1.0f);
-			float armorChanceModifer = GibChance + SelfArmor.DismembermentProtectionChance;
-			if(Severity == DamageSeverity.Max || currentCutSize == BodyPartCutSize.LARGE){armorChanceModifer -= 0.25f;} //Make it more likely that the bodypart can be gibbed in it's worst condition.
-			if(chance >= armorChanceModifer)
+			if (GibChance == 0)
+				return;
+			var armorChanceModifer = GibChance - SelfArmor.DismembermentProtectionChance;
+			if (Severity == DamageSeverity.Max || currentCutSize == BodyPartCutSize.LARGE)
+			{
+				armorChanceModifer += 25; //Make it more likely that the bodypart can be gibbed in it's worst condition.
+			}
+			
+			var chance = UnityEngine.Random.RandomRange(0, 100);
+			if(chance < armorChanceModifer)
```

Comment:
```suggestion
			var chance = UnityEngine.Random.RandomRange(0, 101);
			if(chance < armorChanceModifer)
```
RandomRange for ints is exclusive for max value